experimental/gittuf/repository_test.go

```diff
@@ -13,21 +13,38 @@ import (
 )
 
 func TestLoadRepository(t *testing.T) {
-	tmpDir := t.TempDir()
-	gitinterface.CreateTestGitRepository(t, tmpDir, false)
+	t.Run("load with no repo, unsuccessful", func(t *testing.T) {
```

Comment:
the tests in this file are fine, I suggest we merge this PR with just those and circle back on the policy_test.go ones?